builder/common/generated_data.go

```diff
@@ -0,0 +1,16 @@
+package common
+
+import "github.com/hashicorp/packer/helper/multistep"
+
+type GeneratedData struct {
+	Data  map[string]interface{}
+	State multistep.StateBag
```

Comment:
I think it would be nice also to comment on why we are using two sets here ( aren't they doing the same thing ? )

Reply:
I'm making it simple by removing the Data. You're right, both are the same. I wrote Data first and then I didn't realize it could be simpler! Thank you

Reply:
Okay ! 🙂 👍